.github/workflows/03-ci-rebase-squash.yml

```
@@ -0,0 +1,131 @@
+# Rebase or squash branch
+#
+# Description:
+#   rebase or squash current branch (main->current)
+#   automatically with a comment in the Pull request.
+#
+# Usage:
+#   comment: "@gitbot rebase"  for rebase branch
+#   comment: "@gitbot squash"  for squash branch
+#
+# Requirements:
+#   The author of the comment must meet at least
+#   one requirement:
+#     - Be the owner of the repository
+#     - Be a member of the organization
+#     - Be on the assignees list
+#     - Be the author of the commit
+#
+# Interactions:
+#  👀. Caught comment:
+#   1. When the comment is detected (e.g. @gitbot rebase)
+#      the reaction '👀' will be added to the comment.
+#
+#  👎. Failed:
+#   1. If the author doesn't meet the requirements,
+#      the reaction '👎' will be added to the comment
+#      and the comment will be answered indicating
+#      which has no necessary requirements (See comment below).
+#
+#   2. If the autor meets requirements and the action fails,
+#      the reaction '👎' will be added to the comment.
+
```

You forgot a number sign `#`

```suggestion
#
```